front-end/src/components/BlocksToTime.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import { bnToBn } from '@polkadot/util';
+import BN from 'bn.js';
+import React, { useContext, useState } from 'react';
+import { useEffect } from 'react';
+import { Popup } from 'semantic-ui-react';
+import { ApiContext } from 'src/context/ApiContext';
+import { chainProperties } from 'src/global/networkConstants';
+import blockToTime from 'src/util/blockToTime';
+import getNetwork from 'src/util/getNetwork';
+
+interface Props {
+	blocks: number | BN;
+	className?: string
+}
+
+const BlocksToTime = ({ blocks, className }:Props ) => {
+	const network = getNetwork();
+	const { api, apiReady } = useContext(ApiContext);
+	const DEFAULT_TIME = bnToBn(chainProperties?.[network]?.blockTime);
+	const [blocktime, setBlocktime] = useState(DEFAULT_TIME);
+
+	useEffect(() => {
+		if (!api) {
+			console.error('polkadot/api not set');
+			return;
+		}
+
+		if (!apiReady) {
+			console.error('api not ready');
+			return;
+		}
+
+		let unsubscribe: () => void;
+
+		setBlocktime(api?.consts.babe?.expectedBlockTime);
+		return () => unsubscribe && unsubscribe();
+	}, [ api, apiReady]);
+
+	const popupStyle = {
+		fontSize: '1.2rem'
```

Comment:
See reply above.

Reply:
I'm sure there are workarounds. What prevents us from having this popup inside a span or div that itself is styled?

Reply:
Thanks, that worked!

Reply:
Edit: this gives a warning 
```
The component styled.div with the id of "sc-pkfPC" has been created dynamically.
You may see this warning because you've called styled inside another component.
To resolve this only create new StyledComponents outside of any render method and function component.
```